app/models/katello/ping.rb

```diff
@@ -28,7 +28,8 @@ def ping(services: SERVICES, capsule_id: nil)
         # set overall status result code
         result = {:services => result}
         result[:services].each_value do |v|
-          result[:status] = v[:status] == OK_RETURN_CODE ? OK_RETURN_CODE : FAIL_RETURN_CODE
+          #result[:status] = v[:status] == OK_RETURN_CODE ? OK_RETURN_CODE : FAIL_RETURN_CODE
```

Comment:
Is there a reason for keeping the commented-out line there? Also am I reading it wrong or are you just replacing all the collected result statuses with `FAIL_RETURN_CODE`?